Refactored Rui, tests, and imports. Removed extra init in Rui.

mrw8 opened the pull request:
Made Rui more pythonic without changing the functionality of the class by leveraging Enums. The a_or_r field was converted to an Enum instead of a String in order to reflect the invariant nature of the statuses and the potential expansion of more statuses. I then made the corresponding changes in the other modules to reflect the shift to the enum. 

Imports were then changed to allow for pytest to be run from the project's root directory.

Commit:
Refactored Rui, tests, and imports. Removed extra init in Rui.

hoganwr commented:
I'm getting an error when I try to build:

(rt2) ABC-123 ~/devel/repos/software/rt2% python3 -m build   
* Creating isolated environment: venv+pip...
* Installing packages in isolated environment:
  - setuptools
* Getting build dependencies for sdist...
* Building sdist...
Traceback (most recent call last):
  File "/Users/asdf/.local/share/virtualenvs/rt2-CDbAD8fb/lib/python3.12/site-packages/pyproject_hooks/_in_process/_in_process.py", line 373, in <module>
    main()
  File "/Users/asdf/.local/share/virtualenvs/rt2-CDbAD8fb/lib/python3.12/site-packages/pyproject_hooks/_in_process/_in_process.py", line 357, in main
    json_out["return_val"] = hook(**hook_input["kwargs"])
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asdf/.local/share/virtualenvs/rt2-CDbAD8fb/lib/python3.12/site-packages/pyproject_hooks/_in_process/_in_process.py", line 326, in build_sdist
    return backend.build_sdist(sdist_directory, config_settings)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/var/folders/rf/r6l5cf990b116sfn7dzdj39c0000gq/T/build-env-g0htosjx/lib/python3.12/site-packages/setuptools/build_meta.py", line 426, in build_sdist
    return self._build_with_temp_dir(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/var/folders/rf/r6l5cf990b116sfn7dzdj39c0000gq/T/build-env-g0htosjx/lib/python3.12/site-packages/setuptools/build_meta.py", line 399, in _build_with_temp_dir
    result_basename = _file_with_extension(tmp_dist_dir, result_extension)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/var/folders/rf/r6l5cf990b116sfn7dzdj39c0000gq/T/build-env-g0htosjx/lib/python3.12/site-packages/setuptools/build_meta.py", line 123, in _file_with_extension
    raise ValueError(
ValueError: No distribution was found. Ensure that `setup.py` is not empty and that it calls `setup()`.

Commit:
Removed setup.py, as it is not currently being used and causing errors. Fixed imports to rtt to be src.rtt

mrw8 commented:
The build errors should be fixed now, as I removed setup.py. I fixed a few more imports after this, so it should be able to build and run on pytest now.

hoganwr reviewed (approved):
Everything looks good!
